docs/led.md

```diff
@@ -45,6 +45,7 @@ All of these values can be set by default for when the keyboard boots.
 from kmk.extensions.led import AnimationModes
 led_ext = LED(
     led_pin=led_pin,
+    brightness=100,
```

Comment:
Why do the docs say something, but then do something different? This number changes